src/components/TaskList.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { fetchData, sortDateTime } from "./../utils/utils";
+import AddTaskButton from "./../components/AddTaskButton";
+import {
+	MainCont,
+	TaskBox,
+	DateText,
+	Description,
+	TaskTitle,
+	TaskCont,
+	CheckBox,
+	Icon,
+} from "../styled-components/reusables";
+
+import tick from "./../images/tick.svg";
+
+export default function TaskList({ url }) {
+	const [tasks, setTasks] = useState(null);
+
+	useEffect(() => {
+		resolveTaskData();
+	}, []);
+
+	function resolveTaskData() {
+		fetchData(`${url}`)
+			.then((result) => setTasks(result))
+			.catch((error) => error);
```

Comment:
The error handling is missing - you catch the error but there is no information show to the user in case there is one. Do you have an idea on how this could be implemented?